src/PageContainer/component.js

```diff
@@ -35,10 +43,24 @@ export default class PageContainer extends Component {
     this.preparePage(this.props, this.context)
   }
 
+  componentDidMount() {
+    this.catchInternalLink()
+  }
```

Comment:
https://facebook.github.io/react/docs/component-specs.html tells me that this should be in componentWillUnmount. But not sure if that make sense to stop listening since removed (dropped) from the dom. The GC should handle this right?